src/OutstreamPlayer.js

```diff
@@ -41,8 +41,16 @@ export default class OutstreamPlayer {
         this.player = new PlayerFactory();
         logger.log("Player object: " + JSON.stringify(this.player));
 
-        this.player.generatePlayerConfig(this.bid, this.elementId, this.config);
-        this.insertPlayer();
+        if (this.bid.hasOwnProperty('vastUrl') && !this.bid.hasOwnProperty('vastXml') && !this.bid.hasOwnProperty('ad')) {
```

Comment:
So before the player had to fetch the vastXML? Or did this not work at all?

Reply:
This is just a check for the existence of the actual vastXml in the bidder response.
I came across it using openX, where they only return vastUrl and no vastXML, hence the player could not load the video.

I thought that integrating the vastUrl -> vastXml solution was more appropriate here, instead in bidder adapter code, so 
any bidder with the same attitude would be handled properly.

The best solution would be to have all bidders return a vastXml and skip "hacks" like these.

Reply:
> any bidder with the same attitude would be handled properly.

I quite enjoyed this phrase :joy: 

> The best solution would be to have all bidders return a vastXml and skip "hacks" like these.

I hear you. Oustream is so... fragmented.